repository/src/test/java/org/alfresco/encryption/EncryptionTests.java

```diff
@@ -101,6 +101,18 @@
 
 	private UserTransaction tx;
 
+	static SecureRandom random;
+	static {
+		try
+		{
+			random = SecureRandom.getInstance("SHA1PRNG");
+			random.setSeed(System.nanoTime());
+		} catch (NoSuchAlgorithmException e)
+		{
+			e.printStackTrace();

```

Comment:
Throw some kind of `RuntimeException` here. not throwwing exception here will lead to the NPEs later on.